requirements.txt

```diff
@@ -4,8 +4,8 @@ chardet==3.0.4
 coverage==4.5.3
 easyimap==0.6.3
 EasyProcess==0.2.5
+kbhff==0.0.0
 more-itertools==7.0.0
-pkg-resources==0.0.0
 pluggy==0.9.0
```

Comment:
It seems there's [a CVE](https://nvd.nist.gov/vuln/detail/CVE-2019-11324) recommending to upgrade urllib to `urllib3>=1.24.2`.